multiraft/rpc.go

```diff
@@ -0,0 +1,25 @@
+// Copyright 2015 The Cockroach Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+// implied. See the License for the specific language governing
+// permissions and limitations under the License. See the AUTHORS file
+// for names of contributors.
+//
+// Author: Tamir Duberstein (tamird@gmail.com)
+
+package multiraft
+
+// GetUser implements userRequest.
+// Raft messages are always sent by the node user.
+func (m *RaftMessageRequest) GetUser() string {
+	// TODO(marc): we should use security.NodeUser here, but we need to break cycles first.
```

Comment:
Is importing `security` from `multiraft` a cycle? I thought that was only a problem in the `proto` package.